index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
```suggestion
<html lang="ru" dir="ltr">
```

контент сайта же на русском, поэтому тут тоже надо менять 🙂